tests/conftest.py

```diff
@@ -113,3 +113,29 @@ def hydds(flwda, flwdir):
     )
     ds.raster.set_crs(flwda.raster.crs)
     return ds
+
+
+@pytest.fixture
```

Comment:
I think we could only do with a single normalized time series instead of `obsda` en `simda`. Then in the tests one can create a second timeseries based on the first. Which also makes it directly clear how the two are related and thus what the statistic should be.